src/Instrumentation/Event.php

```diff
@@ -96,6 +96,16 @@ public function add_properties( $properties ) {
 		}
 	}
 
+	/**
+	 * Sanitize key.
```

Comment:
A bit of a no-value comment here. How about explaining what we sanitize for?
```suggestion
	 * Sanitize key to use it for an HTTP header label (alphanumeric and dashes/underscores only).
```